protocol/src/common/protocol.ts

```diff
@@ -1275,28 +1343,49 @@ export interface ServerCapabilities<T = LSPAny> {
 	/**
 	 * Workspace specific server capabilities.
 	 */
-	workspace?: {
-		/**
-		 * The server supports workspace folder.
-		 *
-		 * @since 3.6.0
-		 */
-		workspaceFolders?: WorkspaceFoldersServerCapabilities;
-
-		/**
-		* The server is interested in notifications/requests for operations on files.
-		*
-		* @since 3.16.0
-		*/
-		fileOperations?: FileOperationOptions;
-	};
+	workspace?: WorkspaceOptions;
 
 	/**
 	 * Experimental server capabilities.
 	 */
 	experimental?: T;
 }
 
+/**
+ * Information about the server
+ *
+ * @since 3.15.0
+ */
+export interface ServerInfo {
+	/**
+	 * The name of the server as defined by the server.
+	 */
+	name: string;
+
+	/**
+	 * The server's version as defined by the server.
+	 */
+	version?: string;
+}
+
+
+/**
+ * Information about the client
+ *
```

Comment:
We should add @since 3.18.0 own type name or something to indicate the change we did.

Reply:
Same for ServerInfo and all other types we introduce that have already a @since tag